src/protagonist/Test.Helpers/Integration/DlcsDatabaseFixture.cs

```diff
@@ -46,6 +46,7 @@ public DlcsDatabaseFixture()
     /// </summary>
     public void CleanUp()
     {
+        DbContext.Database.ExecuteSqlRaw("CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";");

```

Comment:
might be best to move this to a migration...

Reply:
for context `InsetFromQuery` ultimately converts `Guid.NewGuid()` to `uuid_generate_v4` in postgres, which needs this extension added to work